src/util/createSlug.js

```diff
@@ -1,19 +1,19 @@
 function createSlug(title) {
   let slug = title.toLowerCase().trim();
-  
+
   const sets = [
     {to: 'nodejs', from: /node.js/ }, // Replace node.js
     {to: '-and-', from: /&/ }, // Replace &
-    {to: '-', from: /(\/|_|,|:|;|\\|\ |\.)/g } // Replace /_,:;\. and whitespace
+    {to: '-', from: /([\/_,:;\\ .])/g } // Replace /_,:;\. and whitespace
```

Comment:
nit: The capture group is unnecessary.

nit2: Within a character set, only `\`, `-`, and `]` need to be escaped.

```suggestion
    {to: '-', from: /[/_,:;\\ .]/g } // Replace /_,:;\. and whitespace
```